app/ng-ui-widgets-category/tabs/styling/article.md

```diff
@@ -1,3 +1,7 @@
 > **Note:** - The integration with `nativescript-theme` and the support for custom CSS is currently under development and is on its way.
 
-<snippet id='tabs-theming-css-ng'/>
\ No newline at end of file
+<snippet id='tabs-theming-css-ng'/>
+
+> **Note:** Currently, we can set only the `backgroundColor`, `color`, `fontFamily`, `fontSize`, `fontStyle`, `fontWeight` and `textTrasnform` CSS properties to the `Label` and `Image` components inside the TabStripItem. 
```

Comment:
```suggestion
> **Note:** Currently, we can set only the `backgroundColor`, `color`, `fontFamily`, `fontSize`, `fontStyle`, `fontWeight` and `textTransform` CSS properties to the `Label` and `Image` components inside the TabStripItem. 
```

Reply:
Also, it's customary to name CSS properties with their kebab case variants, e.g. `background-color`, `text-transform` etc.